src/registrar/tests/test_views_domain.py

```diff
@@ -1521,6 +1523,52 @@ def test_federal_agency_submit_blocked(self):
         self.assertNotEqual(self.domain_information.federal_agency, new_value)
 
 
+class TestDomainSuborganization(TestDomainOverview):
+    """Tests the Suborganization page for portfolio users"""
+
+    @less_console_noise_decorator
+    @override_flag("organization_feature", active=True)
+    def test_has_suborganization_field_on_overview_with_flag(self):
+        """Ensures that the suborganization field is visible
+        and displays correctly on the domain overview page"""
+
+        # Create a portfolio
+        portfolio = Portfolio.objects.create(creator=self.user, organization_name="Ice Cream")
+        _suborg = Suborganization.objects.create(portfolio=portfolio, name="Vanilla")
```

Comment:
**Question:** Not a concern or blocker, just more curious -- what is the purpose behind the _ in front of suborg when naming?

Reply:
All variables are throw away since they will only exist the defined scope. I was always taught the "_" is only supposed to be used on variables that need to be captured but will likely never be used. _ in that case will not be used and is used to just break apart the tuple. You do use _suborg later so I don't think the _ provides added meaning here